tensorflow/lite/experimental/micro/examples/micro_recognition/disco/display_util.cc

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/lite/experimental/micro/examples/micro_recognition/disco/display_util.h"
+
+#include "LCD_DISCO_F746NG/LCD_DISCO_F746NG.h"
+
+#include <stdint.h>
+
+LCD_DISCO_F746NG lcd;
+
+extern "C" {
+// defined in stm32746g_discovery_camera.c
+extern DCMI_HandleTypeDef hDcmiHandler;
+void DCMI_IRQHandler(void) { HAL_DCMI_IRQHandler(&hDcmiHandler); }
+void DMA2_Stream1_IRQHandler(void) {
+  HAL_DMA_IRQHandler(hDcmiHandler.DMA_Handle);
+}
+}
+
+static char lcd_output_string[50];
+
+void init_lcd() { lcd.Clear(LCD_COLOR_WHITE); }
+
+void display_image_rgb888(int x_dim, int y_dim, uint8_t* image_data, int x_loc,
+                          int y_loc) {
+  for (int y = 0; y < y_dim; y++) {
+    for (int x = 0; x < x_dim; x++) {
+      int pix_loc = (y * x_dim + x) * 3;
```

Comment:
I'd suggest removing `pix_loc`.
```cpp
for (int y = 0; y < y_dim; ++y) {
  for (int x = 0; x < x_dim; ++x, image_data += 3) {
     auto r = image_data[0];
     auto g = image_data[1];
     auto b = image_data[2];
     .....
  }
}
```

In addition, `++y` and `++x` is preferred over `y++` and `x++`.